ivy/functional/frontends/paddle/search.py

```diff
@@ -33,6 +33,15 @@ def argsort(x, /, *, axis=-1, descending=False, name=None):
     return ivy.argsort(x, axis=axis, descending=descending)
 
 
+@with_supported_dtypes(
+    {"2.5.1 and below": ("int32", "int64", "float32", "float64")},
+    "paddle",
+)
+@to_ivy_arrays_and_back
+def index_sample(x, index):
```

Comment:
the signature of `index_select` does not match the original function's signature [here](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/index_select_cn.html#paddle.index_select)